themes/bootstrap5/js/common.js

```diff
@@ -488,8 +512,8 @@ var VuFind = (function VuFind() {
 
   /**
    * Initialize result page scripts.
-   *
-   * @param {string|Element} _container
+   * @param {string|Element} [_container] The container element to initialize scripts on.
```

Comment:
I think this is non-optional, but please correct me if I'm confused:

```suggestion
   * @param {string|Element} _container The container element to initialize scripts on.
```